docs/get-started/getting-started-databricks.md

```diff
@@ -50,9 +50,12 @@ The number of GPUs per node dictates the number of Spark executors that can run
 4. Databricks makes changes to the runtime without notification.
 
     Databricks makes changes to existing runtimes, applying patches, without notification.
-	[Issue-3098](https://github.com/NVIDIA/spark-rapids/issues/3098) is one example of this.  We run
-	regular integration tests on the Databricks environment to catch these issues and fix them once
-	detected.
+    [Issue-3098](https://github.com/NVIDIA/spark-rapids/issues/3098) is one example of this.  We run
+    regular integration tests on the Databricks environment to catch these issues and fix them once
+    detected.
+5. In Databricks 11.3, an incorrect result is returned for window frames defined by a range in case 
+   of DecimalTypes with precision greater than 38. There is a bug filed in Apache Spark for it 
+   [here](https://issues.apache.org/jira/browse/SPARK-41793)
```

Comment:
I believe we are returning the correct answer in RAPIDS Spark.  If so, we can say:
```suggestion
5. In Databricks 11.3, an incorrect result is returned for window frames defined by a range in case 
   of DecimalTypes with precision greater than 38. There is a bug filed in Apache Spark for it 
   [here](https://issues.apache.org/jira/browse/SPARK-41793), whereas when using the plugin the 
   correct result will be returned.  
```